specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the RecoveryServicesBackup API.
 title: Recovery Services Backup Client
 description: Open API 2.0 Specs for Azure RecoveryServices Backup service
 openapi-type: arm
-tag: package-2024-04
+tag: package-preview-2024-07
```

Comment:
We do not change this tag for preview version. 
Ref
https://github.com/Azure/azure-rest-api-specs/pull/20934/files#diff-278e6b2057b04d09991cfb9fedfda4baf53f7f073d14b022ade16da0ee7abf57